Sources/x10/swift_bindings/XLATensor.swift

```diff
@@ -1069,3 +1069,7 @@ extension XLATensor {
     }
   }
 }
+
+public func PrintX10Metrics() {
+  x10_xla_tensor_wrapper.PrintMetrics()
```

Comment:
I think that you can get away with `PrintMetrics()` but I may be mistaken.

Reply:
Done.